pkg/setup/binary_cni.go

```diff
@@ -20,6 +20,7 @@ func (e *Environment) extractCNI() error {
 	output := string(b)
 	if err != nil {
 		glog.Errorf("Cannot untar %s, %s: %v", e.binaryCNI.archivePath, output, err)
+		e.binaryCNI.removeArchive()
```

Comment:
Does it make sense to write ` _ = e.binaryCNI.removeArchive()` to indicate that the error is being ignored?

Reply:
@devonboyer cool idea, I was thinking about a comment but I guess this is self explicit.